### DISCUSSION TITLE
custom confirmation emails / kill switch for internal emails

### DISCUSSION THREAD

**Discussion by logemann** [supabase/supabase#11366] [Questions]
Hi,

its easy enough to create your own confirmation emails for various actions but how do i disable the internal email sending of the template string which we can configure in the UI. Of course i dont want to send one custom email and the one provided by supabase. Currently i am using a /dev/null custom email service so that the supabase generated email  just goes to oblivion but i dont want to use a 3rd party service for something like this. 

Is there really no kill-switch for the internal email sending?

**Comment by suamai** [comment]
Any answers on this topic?

**Comment by stefan-girlich** [comment]
@logemann I am in the same situation. Which custom /dev/null email service are you using? I was thinking about https://ethereal.email/ , but exposing your users' email addresses to a third-party is problematic in itself.

**Reply by logemann** [reply]
does supabase still have this problem that you cant just disable emails in the studio? Damn. I stopped using one of the services and build my own one. But i used to use the one you mentioned.

Perhaps its worth pinging supabase support about this issue here on GH.

**Reply by stefan-girlich** [reply]
Thanks for the quick response. Will try Ethereal for now and try to get an update from Supabase on the "off switch" feature. Would you know how to mention ideally?

**Comment by logemann** [comment]
Hey supabase team? Still think it would be a worthwhile feature....
